documents/motions/10.2021.10 - Results Team.md

```diff
@@ -22,3 +22,16 @@ The WCA Results Team is an Advisory Committee of the WCA with the following righ
 3. Management of Rankings
    1. Recognizing and administering the personal, national, continental, and world records per event.
    2. Recognizing and administering the national, continental, and world rankings per event.
+4. All matters relating to Data Protection:
+   1. Creation, implementation, and enforcement of all WCA Data Protection policies:
+      1. Researching regulations globally to ensure the WCA adopts the policy requirements to satisfy all global regulations.
+      2. Engaging with legal council on all matters relating to Data Protection with WCA Board approval.
+      3.  Understanding the scope of which Data Protection policies are required and duly advising the WCA Board.
```

Comment:
There is an excess space at the beginning.

Reply:
Thanks, resolved.